src/finalize.cpp

```diff
@@ -85,7 +85,6 @@ int openmc_finalize()
   settings::material_cell_offsets = true;
   settings::max_particles_in_flight = 100000;
   settings::max_splits = 1000;
-  settings::max_tracks = 1000;
```

Comment:
Was this intentional?

Reply:
Nope, don't what happened here

Reply:
Nope, it was actually not intentional. It's of course meant to be there.